src/database/models/Accommodations.js

```diff
@@ -73,6 +73,7 @@ module.exports = (sequelize, DataTypes) => {
     cost: DataTypes.DOUBLE,
     services: DataTypes.STRING,
     amenities: DataTypes.STRING,
+    status:DataTypes.STRING,
```

Comment:
Let the changes reflect here in the model.

Reply:
thank you @elchroy, I applied the changes